hack/build.sh

```diff
@@ -50,7 +50,11 @@ function squash {
 dirs=${VERSION:-$VERSIONS}
 
 for dir in ${dirs}; do
-  IMAGE_NAME="${BASE_IMAGE_NAME}-${dir//./}-${OS}"
+  if [ "$dir" == "2.6" ]; then
+    IMAGE_NAME=rhscl/rh-mongodb${dir//./}-${OS}
```

Comment:
Name of the rh-mongodb26 image should be named for example rhscl/rh-mongodb26-rhel7 - see https://mojo.redhat.com/docs/DOC-1018140
And it is quite different to openshift/mongodb-24-rhel7.